src/Scripting/Core/Script.cs

```diff
@@ -86,12 +98,30 @@ internal static Script<T> CreateInitialScript<T>(ScriptCompiler compiler, string
         public Script<object> ContinueWith(string code, ScriptOptions options = null) =>
             ContinueWith<object>(code, options);
 
+        #pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
```

Comment:
>#pragma warning disable RS0026 // Do not add multiple p [](start = 8, length = 55)

You can just move this to the top of the file, no need to repeat multiple times.